functions/join.go

```diff
@@ -65,31 +66,25 @@ func Join(flags *viper.Viper) error {
 		flags.Set("apiconn", accessToken.APIConnString)
 	}
 	logger.Log(1, "Joining network: ", flags.GetString("network"))
-	node, newServer, newHost, err := JoinNetwork(flags)
+	node, server, err := JoinNetwork(flags)
 	if err != nil {
 		return err
 	}
+	log.Println("server response to join")
```

Comment:
debug logs?

Reply:
Yes, and for now they should stay.  Added to netclient cleanup